opa/models/res_partner.py

```diff
@@ -0,0 +1,10 @@
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    instructor = fields.Boolean()
+    session_ids = fields.Many2many(string='Sessions', comodel_name='session',
```

Comment:
Break line after parenthesis

Reply:
@antonag32 apply the Black formatter suggestion:

```suggestion
    session_ids = fields.Many2many(
        string='Sessions',
        comodel_name='session',
        relation='session_res_partner_rel',
        column1='res_partner_id',
        column2='session_id',
    )
```

Reply:
Done

Reply:
@antonag32 you didn't apply this suggestion